packages/rum-core/src/domain/rumEventsCollection/action/clickChain.spec.ts

```diff
@@ -105,34 +100,25 @@ describe('createRageClickChain', () => {
   describe('when rage is detected', () => {
     it('discards individual clicks', () => {
       const clicks = [createFakeClick(), createFakeClick(), createFakeClick()]
-      createValidatedRageClickChain(clicks)
+      createValidatedClickChain(clicks)
       clicks.forEach((click) => expect(click.discard).toHaveBeenCalled())
     })
 
     it('uses a clone of the first click to represent the rage click', () => {
       const clicks = [createFakeClick(), createFakeClick(), createFakeClick()]
-      createValidatedRageClickChain(clicks)
+      createValidatedClickChain(clicks)
       expect(clicks[0].clonedClick).toBeTruthy()
       expect(clicks[0].clonedClick?.validate).toHaveBeenCalled()
     })
 
     it('the rage click should have a "rage" frustration', () => {
       const clicks = [createFakeClick(), createFakeClick(), createFakeClick()]
-      createValidatedRageClickChain(clicks)
-      const expectedFrustrations = new Set()
-      expectedFrustrations.add(FrustrationType.RAGE_CLICK)
-      expect(clicks[0].clonedClick?.getFrustrations()).toEqual(expectedFrustrations)
+      createValidatedClickChain(clicks)
+      expect(clicks[0].clonedClick?.validate).toHaveBeenCalledWith([FrustrationType.RAGE_CLICK])
     })
 
-    it('the rage click should contains other clicks frustration', () => {
-      const clicks = [createFakeClick(), createFakeClick(), createFakeClick()]
-      clicks[1].addFrustration(FrustrationType.DEAD_CLICK)
-      createValidatedRageClickChain(clicks)
-      expect(clicks[0].clonedClick?.getFrustrations().has(FrustrationType.RAGE_CLICK)).toBe(true)
-    })
```

Comment:
[DRAFT] this test has been removed, because we don't copy frustrations to the rage click anymore. I'll write tests focused on the `finalizeClicks` once we agree on this design.